lib/check_board.rb

```diff
@@ -5,23 +5,63 @@ def initialize
     @score = ''
   end
 
-  def execute(*board)
-    @check_score = board.reduce
+  def execute(board)
```

Comment:
Instead of passing the `board` as a parameter, you could make `view_board` a dependency (pass it into the initializer).

This would simplify the code in your future UI and in your acceptance test code. 

(Acceptance tests are just another UI)